scripts/globals/magic.lua

```diff
@@ -1428,8 +1428,71 @@ function canOverwrite(target, effect, power, mod)
     return true;
 end
 
-function doElementalNuke(V,M,caster,spell,target,hasMultipleTargetReduction,resistBonus)
-	return doNuke(V,M,caster,spell,target,hasMultipleTargetReduction,resistBonus,ELEMENTAL_MAGIC_SKILL,MOD_INT);
+function doElementalNuke(caster, spell, target, spellParams)
+    local DMG = 0;
+    local V = 0;
+    local M = 0;
+    local dINT = caster:getStat(MOD_INT) - target:getStat(MOD_INT);
+    local hasMultipleTargetReduction = spellParams.hasMultipleTargetReduction; --still unused!!!
+    local resistBonus = spellParams.resistBonus;
+    local mDMG = caster:getStat(MOD_MAGIC_DAMAGE);
+
+    if(mDMG == nil) then
+        mDMG = 0;
+    end		
+
+    --[[
+            Calculate base damage:       
+            D = mDMG + V + (dINT × M)
+            D is then floored
+            For dINT reduce by amount factored into the V value (example: at 134 INT, when using V100 in the calculation, use dINT = 134-100 = 34)             
+        ]]
+    if(dINT <= 0) then 
+        DMG = V + dINT; --unsure how to handle dINT <= 0!!!

```

Comment:
For dINT < 0, reduced M values apply. With a negative dINT, calculated value for D will be lower than V, potentially even 0.